src/App.tsx

```diff
@@ -1,24 +1,151 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { Filter } from './types/Filter';
+import { Error } from './types/Error';
+import { ErrorMessage } from './components/ErrorMessage';
+import { addTodos, deleteTodos, getTodos } from './api/todos';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
 
-const USER_ID = 0;
+const USER_ID = 27;
+
+export const filterTodos = (currentTodos: Todo[], query: Filter) => {
+  return currentTodos.filter(todo => {
+    switch (query) {
+      case Filter.All:
+        return todo;
```

Comment:
This case looks redundant because you already return the same thing in the default case